ray-operator/test/e2e/support.go

```diff
@@ -125,7 +125,7 @@ func headPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyConfigura
 				WithName("ray-head").
 				WithImage(GetRayImage()).
 				WithPorts(
-					corev1ac.ContainerPort().WithName("gcs").WithContainerPort(6379),
+					corev1ac.ContainerPort().WithName("gcs-server").WithContainerPort(6379),
```

Comment:
Not necessarily in this PR, but would you mind using constant variables instead of hard-coded strings?